templates/amq-broker-71-statefulset-clustered-drain.yaml

```diff
@@ -0,0 +1,334 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: amq-broker-71-persistence-with-clustering
+  xpaas: 1.4.12
+message: A new messaging service has been created in your project. It will handle the protocol(s) "${AMQ_PROTOCOL}". The username/password for accessing the service is ${AMQ_USER}/${AMQ_PASSWORD}.
+metadata:
+  annotations:
+    description: Application template for JBoss AMQ brokers. These can be deployed as standalone or in a mesh. This template doesn't feature SSL support.
+    iconClass: icon-amq
+    openshift.io/display-name: JBoss AMQ Broker 7.1 (Ephemeral, no SSL)
+    openshift.io/provider-display-name: Red Hat, Inc.
+    tags: messaging,amq,jboss
+    version: 1.4.9
```

Comment:
@vbusch Not sure if you want to update this before the PR gets merged.

Reply:
Updating it now